CMakeLists.txt

```diff
@@ -46,6 +46,11 @@ if(USE_CUDA AND NOT USE_OLDCMAKECUDA)
         (${CMAKE_VERSION} VERSION_GREATER "3.9.0") OR (${CMAKE_VERSION} VERSION_EQUAL "3.9.0")
       )
     )
+    
+    set(__cuda_toolset "7.5" "8.0" "9.0")
+    set(CUDA_TOOLSET "8.0" CACHE STRING "Select CUDA Version.")
```

Comment:
Why doesn't the other method work? It appears to be working on our visual studio  CI builds.  It's not fair to require the user to set CUDA_TOOLSET and to keep updating __cuda_toolset (9.1 is available now).

Reply:
sorry for later,if not set CUDA_TOOLSET,it will use the latest version cuda.
If you install multiple cuda versions at the same time, you need CUDA_TOOLSET to select the version.